src/main/java/net/mcreator/ui/blockly/BlocklyJavascriptBridge.java

```diff
@@ -276,6 +276,10 @@ public void addExternalTrigger(ExternalTrigger external_trigger) {
 		return new Gson().toJson(ext_triggers, Map.class);
 	}
 
+	@SuppressWarnings("unused") public String[] getCustomEntityDataList(String entityName) {

```

Comment:
This new method redirects to the same thing as `getReadableListOfWithValue("entitydata", value)`, I think it should be replaced with this reference.
Also, at this point, the two arrays the `arrayToBlocklyDropDownArrayWithReadableNames()` function in the new extension definition gets have the same contents. Just noting this one.